vshard/replicaset.lua

```diff
@@ -421,6 +525,26 @@ local function replica_call(replica, func, args, opts)
     assert(opts and opts.timeout)
     replica.activity_ts = fiber_clock()
     local conn = replica.conn
+    if opts.is_async then
```

Comment:
Need a test that for an `is_async` request the call fails instantly without any yields if the connection is not ready.

Reply:
https://github.com/tarantool/vshard/issues/456